### ISSUE TITLE
[Bug] Availibility zones change depending on region selection order during Universe creation

### ISSUE THREAD

**Issue by adil246** [yugabyte/yugabyte-db#8003]
**Description**
Az's disappear from the list when a new  _region_ is added. This issue appears for the on-prem config provider. 
If the order is changed when adding regions(for example East first and then Central), the AZ's appear properly. 
 
**To Reproduce**
1. Create Universe using an on-prem config provider.
2. Add the first _Region_  with Nodes 3 and RF 3.
3. Then add the second _Region_  and you will notice one of the AZ's disappear.
4. When you click _Add Zone_, it appears. 

**Reference ticket**
1081

**Comment by adil246** [comment]
cc @skahler-yuga Let me know if anything else has to be added.

**Comment by streddy-yb** [comment]
hi @mjoshi0923 - please check if this issue reproduces for other providers as well (AWS, GCP, etc).

**Comment by mjoshi0923** [comment]
Sure

**Comment by mjoshi0923** [comment]
@adil246 Do we have this issue? This was resolved long back, it's duplicate issue I guess.

**Comment by adil246** [comment]
@mjoshi0923 Yes, I do not see the issue now in 2.6.3 and AZ’s placement info is not switching to other zones when additional regions are added. I tested with a couple of regions adding them consecutively in google cloud and placement info stayed the same. I could go back and select Az’s based on preference after adding all regions. This can be closed. Thanks for the follow-up.
